cpp/include/cudf/io/orc.hpp

```diff
@@ -394,6 +407,18 @@ class orc_reader_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Set whether to ignore writer timezone in the stripe footer.
+   *
+   * @param ignore Boolean value to enable/disable ignoring writer timezone
+   * @return this for chaining
+   */
+  orc_reader_options_builder& set_ignore_timezone_in_stripe_footer(bool ignore)
```

Comment:
It seems that other setters don't have the `set_` prefix thus let's remove it for consistency.

```suggestion
  orc_reader_options_builder& ignore_timezone_in_stripe_footer(bool ignore)
```